### ISSUE TITLE
Positioning for sidebar content

### ISSUE THREAD

**Issue by marktaylor46** [gantry/gantry5#788]
Following the release of Requiem template and discussion with @mahagr and @kat05 I am raising this issue at their suggestion.

In Requiem we introduced a "fixedside" section to provide a floating social icons bar. We also provided the facility to choose on which side the bar would reside. But the positioning setting has been added to the section settings on the "style" page and I think it would be more appropriate if this were to be added to the section settings on the layout. Indeed, this is exactly what we did for the offcanvas side slider. I think we should be consistent in our approach to this settings.

As we provide more templates we will inevitably provide other fixed or floating bars (maybe even a top/bottom one) and it would seem to make sense to tackle this now.

@mahagr indicated that it had to be done in Requiem that way as it was "tricky" to implement. So, in short, my suggestion is that we make the necessary changes to Gantry 5 to make this easier to implement in the future in a consistent manner.

![2015-09-03_10-58-05](https://cloud.githubusercontent.com/assets/11683218/9655969/e41db5ce-522c-11e5-99ce-7cfed05c0e5e.png)
![2015-09-03_10-59-19](https://cloud.githubusercontent.com/assets/11683218/9655973/e7964ee6-522c-11e5-9c33-2d0dfe1efcb9.png)
![2015-09-03_10-59-47](https://cloud.githubusercontent.com/assets/11683218/9655975/eb0291ac-522c-11e5-952a-1a067921526c.png)

**Comment by mahagr** [comment]
Related to layout changes we're going to make in 5.2

**Comment by w00fz** [comment]
Is this possible now? I'd think a proper way of dealing with this is to be able to extend section.yaml for specific ones (in this case, fixedside) and the ability to add new options. Would also require the developer to tweak the twig in order to render accordingly.

**Comment by mahagr** [comment]
Yes, this requires us to extend `section.yaml` in for example `section-header.yaml` to add options directly into there. Its easy to try to load the more specific file first, but to make this to work, we also need my code from `feature/blueprints` repository, which is still under works.
